### PR TITLE
Adding TS metrics.

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

**Comment by k-jamroz** [comment]
@blazember could you have a look if the descriptions make sense?

**Review by k-jamroz — APPROVED** [review]

File: docs/modules/ROOT/pages/list-of-metrics.adoc [hunk 1]
```diff
@@ -1570,6 +1571,207 @@ with the master.
 |count
 |Number of started transactions
 
+|`tstore.device.freeSpace`
+|bytes
+|Amount of free space in the device directory
+
+|`tstore.device.maxSpace`
+|bytes
+|Amount of total space in the device directory
```
Comment (blazember) [inline docs/modules/ROOT/pages/list-of-metrics.adoc:1580]: Isn't this the maximum usage of the directory?

**Review by blazember — APPROVED** [review]
Looks good, just one comment on what we may want to clarify.

File: docs/modules/ROOT/pages/list-of-metrics.adoc (hunk 1 above)
Comment (k-jamroz) [reply docs/modules/ROOT/pages/list-of-metrics.adoc:1580]: no, it is calculated as `File.getTotalSpace()` for `baseDir` which in turn
> Returns the size of the partition named by this abstract pathname

So it is size of the partition/disk not a max space that was ever used by the Hybrid Log files in given device.

Comment (Serdaro) [reply docs/modules/ROOT/pages/list-of-metrics.adoc:1580]: I also thought (from the metric name) it'd be the maximum. I am guessing changing the metric name is a bit complicated right (like `tstore.device.totalSpace`)?

**Pull request merged**